service_worker.js

```diff
@@ -317,3 +321,35 @@ function logStorageChange(changes, area) {
   }
 }
 chrome.storage.onChanged.addListener(logStorageChange);
+
+
+async function clearOldCacheBackground(tabId) {
+  if (!(await doesTabExist(tabId))) {
+    chrome.storage.local.remove([tabId]);
+  };
+}
+
+async function clearOldCache() {
+  const now = Date.now();
+  chrome.storage.local.get(null, results => {
```

Comment:
you mentioned `session` storage instead of `local` in https://github.com/GoogleChrome/web-vitals-extension/issues/83#issuecomment-1638688991. That still seems like a good idea?

Reply:
Did consider it, but quite like the fact the web vitals survive a restart, and we need to clear the cache anyway.